types/luaparse/index.d.ts

```diff
@@ -43,6 +43,12 @@ export interface Options {
      * Note: setting luaVersion: 'LuaJIT' currently does not enable this option; this may change in the future.
      */
     extendedIdentifiers: false;
+        /** 
```

Comment:
This comment needs to be aligned correctly. I believe this is what the CI failures are for aswell.